swc_registry/get_entry_info.py

```
@@ -64,7 +64,10 @@ def _swc_content(self):
     @property
     def _content(self):
         entries = self._swc_content
-        current_entry = entries.get(self.swc_id, {})
+        current_entry = entries.get(self.swc_id, None)
+        if current_entry is None:
+            print("The content by entered id is not found")
```

You should raise a custom exception here, something like `SWCDoesNotExist()`